src/general/utils/MathUtils.cs

```diff
@@ -131,4 +132,39 @@ public static int PositiveModulo(this int val, int mod)
         double standardDeviation = Math.Sqrt(sumOfSquares / count - average * average);
         return (average, standardDeviation);
     }
+
+    /// <summary>
+    ///   How far in a given direction a set of points can reach from a reference point
+    /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     This can be useful for finding the edge of a cell from a list of its organelle positions, for instance
+    ///   </para>
+    /// </remarks>
+    public static float GetMaximumDistanceInDirection(Vector3 direction, Vector3 referencePoint,
+        List<Vector3> listOfPoints)
```

Comment:
```suggestion
        IEnumerable<Vector3> listOfPoints)
```
So here this would be modified like this.